sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -234,7 +234,7 @@ apple_iap_subscriptions AS (
     CAST(NULL AS TIMESTAMP) AS canceled_at,
     CAST(NULL AS TIMESTAMP) AS cancel_at,
     CAST(NULL AS BOOL) AS cancel_at_period_end,
-    IF(end_time < CURRENT_DATE, end_time, NULL) AS ended_at,
+    IF(end_time < TIMESTAMP(CURRENT_DATE), end_time, NULL) AS ended_at,
```

Comment:
this fixes a bug introduced in #2142 where you can't use `<` between `TIMESTAMP` and `DATE`